### PR TITLE
fix(MeetingsAdapter): export MediaPermissions

### PR DESCRIPTION
**Pull request by karinasigartau0798**

### CODE REVIEW

**Review by cipak — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 export {default as WebexAdapter} from './WebexAdapter';
 export {default as ActivitiesAdapter} from './ActivitiesAdapter';
-export {default as MeetingsAdapter, MeetingControlState, MeetingState} from './MeetingsAdapter';
+export {
+  default as MeetingsAdapter, MeetingControlState, MeetingState, MediaPermissions,
```
Comment (lalli-flores) [inline src/index.js:4]: can you have each export in their own line?
```suggestion
  default as MeetingsAdapter,
  MeetingControlState,
  MeetingState,
  MediaPermissions,
```

**Comment by lalli-flores** [comment]
can you simplify your commit message to `fix(MeetingsAdapter): export MediaPermissions`? Thanks

File: src/index.js (hunk 1 above)
Comment (karinasigartau0798) [reply src/index.js]: Done.

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**